docs/appguide.rst

```diff
@@ -148,8 +163,8 @@ event notifications and timer callbacks:
   and specify the tick frequency.
 
 Additionally if your application is a game or a similar program that should
-not allow the watch to go to sleep when it is running then it should
-arrange to call :py:meth:`~.Manager.keep_awake` from the application's
+not allow the watch to turn off the screen while running (i.e. go to sleep)
```

Comment:
If you really want to clarify what "go to sleep" means then I can tolerate tasteful replacement of "go to sleep" with "go to sleep or turn of the screen".